src/eko/couplings.py

```diff
@@ -446,24 +446,19 @@ def assert_positive(name, var):
         self.method = method.value
 
         nf_ref = couplings.num_flavs_ref
-        max_nf = couplings.max_num_flavs
         scheme_name = hqm_scheme.name
         self.alphaem_running = couplings.em_running
         self.decoupled_running = False
 
         # create new threshold object
         self.a_ref = np.array(couplings.values) / 4.0 / np.pi  # convert to a_s and a_em
-        self.thresholds = thresholds.ThresholdsAtlas(
-            masses,
-            couplings.scale**2,
-            nf_ref,
-            thresholds_ratios=thresholds_ratios,
-            max_nf=max_nf,
-        )
+        matching_scales = (np.array(masses) * np.array(thresholds_ratios)).tolist()
+        self.thresholds_ratios = list(thresholds_ratios)
```

Comment:
`thresholds_ratios` maybe is already a list given the line above

Reply:
Not really: since somewhere else it was a tuple or an array (don't remember which and where), I downgraded the input to `Iterable[float]`.

This `list` is a temporary compatibility step, that won't be any longer required as soon as we will have enforced clearer typing in `eko`.